CHANGELOG.md

```diff
@@ -14,10 +14,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - fix(sumologicexporter): treat resource attributes as fields for otlp #536
+- fix(cascadingfilterprocessor): prevent overriding metrics in cascading filter processor - add processor tag #539
```

Comment:
Do we want to keep this type of links (i.e. `#539` which wouldn't work outside of Github vs embedded markdown links `[#539][#539]`) ?

@SumoLogic/open-source-collection-team ?

Reply:
I'd personally prefer normal MD links that will work locally as well.